staging/src/k8s.io/api/rbac/v1/types.go

```diff
@@ -118,6 +121,9 @@ type Role struct {
 // RoleBinding references a role, but does not contain it.  It can reference a Role in the same namespace or a ClusterRole in the global namespace.
 // It adds who information via Subjects and namespace information by which namespace it exists in.  RoleBindings in a given
 // namespace only have effect in that namespace.
+// The namespace of the request is taken fromn the request URL's path
+// if present there, otherwise is the empty string; a namespace
+// specified in a field selector is ignored.
```

Comment:
this feels somewhat out of place here... there are multiple ways to trigger a query to the authorization system (for example, a subjectaccessreview, or by making a regular API request)...

Reply:
making sure https://kubernetes.io/docs/reference/access-authn-authz/authorization/ accurately reflects how attributes are determined for API requests, then referencing that and subjectaccessreview docs from here might be better

Reply:
Would this go better in the Subject type, by the namespace field? I think it's fine to explain the difference between cluster and namespaced scopes in multiple places. I think explaining how the information is extracted from an ordinary API request is instructive.